{"name": "test_Astock_F10_briefing_tsException", "status": "failed", "statusDetails": {"message": "AssertionError", "trace": "self = <TestCase.StockQuotation.AstockRelevance.test_Astock_F10briefing.TestAstockF10Briefing object at 0x13032370>\n\n    @allure.story('F10简况_ts为异常')\n    def test_Astock_F10_briefing_tsException(self):\n        response = zhuorui('A股', 'F10简况_ts为异常')\n        # print(response.text)\n>       assert_data(response, '000000', 'ok')\n\nTestCase\\StockQuotation\\AstockRelevance\\test_Astock_F10briefing.py:37: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nresponse = <Response [200]>, code = '000000', msg = 'ok'\n\n    def assert_data(response: object, code: object, msg: object) -> object:\n>       assert code == response.json().get(\"code\")\nE       AssertionError\n\nCommon\\assertapi.py:2: AssertionError"}, "start": 1594175464473, "stop": 1594175464507, "uuid": "6411f414-adf6-461f-bf08-f8dc1c24ce1e", "historyId": "cfcc2dc5be621a0efe8e62819157edd3", "testCaseId": "25d366c93b3219056f9cb83a06e2fa84", "fullName": "TestCase.StockQuotation.AstockRelevance.test_Astock_F10briefing.TestAstockF10Briefing#test_Astock_F10_briefing_tsException", "labels": [{"name": "feature", "value": "A股"}, {"name": "story", "value": "F10简况_ts为异常"}, {"name": "parentSuite", "value": "TestCase.StockQuotation.AstockRelevance"}, {"name": "suite", "value": "test_Astock_F10briefing"}, {"name": "subSuite", "value": "TestAstockF10Briefing"}, {"name": "host", "value": "DESKTOP-7K7NA43"}, {"name": "thread", "value": "11296-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "TestCase.StockQuotation.AstockRelevance.test_Astock_F10briefing"}]}